FreeRTOS-Plus/Demo/AWS/Mqtt_Demo_Helpers/mqtt_demo_helpers.c

```diff
@@ -178,8 +178,13 @@
  * in the link below.
  * https://aws.amazon.com/blogs/iot/mqtt-with-tls-client-authentication-on-port-443-why-it-is-useful-and-how-it-works/
  */
-#define AWS_IOT_MQTT_ALPN                "\x0ex-amzn-mqtt-ca"
+#define AWS_IOT_ALPN_MQTT_CA_AUTH        "x-amzn-mqtt-ca"
```

Comment:
Collecting the ALPN strings in one place is a good idea.  I think the selection of the ALPN string should also be done here instead of the #ifdef later in the code (line 358).  By collecting the decision to this place, future code changes to add ALPN's can be completely made in a single place.  Additionally, checking the code to see which choice has been can be done close to the definitions of the choices.